### ISSUE TITLE
Provide public access to RpcHelper

### ISSUE THREAD

**Issue by robert4os** [google/identity-toolkit-java-client#21]
Hi,

I noticed that 'password-users' do not have a verified email. In the forum I saw a suggestion to implement it by getting a oobCode and validating the email ourselves. So I managed to do this, but I had to 'clone' RpcHelper class to make the following requests:

1) Getting the oobCode. RpcHelper.getOobCode is public but it is not exposed through the GitkitClient class.
>>
String oobCode=rpcHelper.getOobCode(params).getString("oobCode");
<<

2) Set the email verification status to 'confirmed', however rpcHelper.invokeGoogle2LegOauthApi has private access
>>
rpcHelper.invokeGoogle2LegOauthApi("setAccountInfo", params));
<<

Thus my suggestion:

Since there are a number of git-API calls not supported currently it would be extremely useful to get 
1) Access to the rpcHelper via gitkitClient.getRpcHelper();
2) Public access to rpcHelper.invokeGoogle2LegOauthApi()
3) Public access to rpcHelper.invokeGitkitApi()

Thank you and best regards - Robert

**Comment by dereksalama** [comment]
I think the right thing to do here is to properly support the verify email functionality as we do for [go](https://github.com/google/identity-toolkit-go-client/blob/master/gitkit/gitkit.go#L309).

**Comment by robert4os** [comment]
I agree, it certainly is the proper way in the long run but meanwhile making RpcHelper more accessible would be extremely helpful. So developers could implement getPublicKeys, resetPassword, setAccountInfo and verifyPassword if they need it. Btw, the toolkit is very useful, many thanks!

**Comment by robert4os** [comment]
Would be obsolete with pull request https://github.com/google/identity-toolkit-java-client/pull/37
